pkg/detectors/aws/aws_test.go

```diff
@@ -17,6 +17,8 @@ import (
 	"github.com/trufflesecurity/trufflehog/v3/pkg/pb/detectorspb"
 )
 
+var unverifiedSecretClient = common.ConstantResponseHttpClient(403, `{"Error": {"Code": "InvalidClientTokenId"} }`)
```

Comment:
Are the values here actually capitalized in the wild?

Reply:
they sure are!

Reply:
although it would be a good idea to do the comparison case-insensitively